components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/APIThrottleHandler.java

```diff
@@ -564,6 +564,7 @@ private boolean doRoleBasedAccessThrottling(MessageContext synCtx, Configuration
 
                     //check for the permission for access
                     if (info != null && !info.isAccessAllowed()) {
+                        log.info("Exceeded the allocated quota in Application level.");
```

Comment:
Why this is info?? in the production systems these logs will be fill up the log file